<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemsWidget</class>
 <widget class="QWidget" name="ItemsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="_3">
     <item row="0" column="0">
      <widget class="QLineEdit" name="SearchLine_"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="SearchType_">
       <item>
        <property name="text">
         <string>Fixed string</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wildcard</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Regexp</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="CaseSensitiveSearch_">
       <property name="text">
        <string>Case sensitive</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="CategoriesLayout_">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTreeView" name="Items_">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="LeechCraft::Plugins::Aggregator::SelectableBrowser" name="ItemView_" native="true"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LeechCraft::Plugins::Aggregator::SelectableBrowser</class>
   <extends>QWidget</extends>
   <header>selectablebrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
